Composer/packages/extensions/visual-designer/src/utils/prefixOperation.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function addEqualPrefix(first: string, second: string): string {
```

Comment:
Since we just move from = to :, do we still need this function?

Reply:
> Since we just move from = to :, do we still need this function?

revert code for the new logic